src/api/ledger/accountinfo.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const utils = require('./utils');
+const removeUndefined = require('./parse/utils').removeUndefined;
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+
+function formatAccountInfo(response) {
+  const data = response.account_data;
+  return removeUndefined({
+    sequence: data.Sequence,
+    xrpBalance: utils.common.dropsToXrp(data.Balance),
```

Comment:
There's also things like TransferRate (very likely want that), and the EmailHash (or whatever it was called, might be deprecated)